__init__.py

```diff
@@ -643,29 +648,38 @@ def _get_speakable_timer_details(self, timers: List[CountdownTimer]) -> str:
 
         return timer_names
 
-    def display_timers(self):
+    def _show_gui(self):
+        """Update the device's display to show the status of active timers.
+
+        Runs once a second via a repeating event to keep the information on the display
+        accurate.
+        """
+        if self.gui.connected:
+            self._update_gui()
+            if self.platform == MARK_II:
+                page = "timer_mark_ii.qml"
+            else:
+                page = "timer_scalable.qml"
+            self.gui.show_page(page, override_idle=True)
+
+    def update_display(self):
         """Update the device's display to show the status of active timers.
 
         Runs once a second via a repeating event to keep the information on the display
         accurate.
         """
         if self.gui.connected:
-            self._display_timers_on_gui()
+            self._update_gui()
         elif self.platform == MARK_I:
             self._display_timers_on_faceplate()
 
-    def _display_timers_on_gui(self):
+    def _update_gui(self):
         """Display active timers on a device that supports the QT GUI framework."""
         timers_to_display = self._select_timers_to_display(display_max=4)
```

Comment:
Shouldn't display_max come from a config file?

Reply:
Which config file should it come from?  Skill configs don't belong in the core config.  It's not a user-defined setting.  It's defined by the screen design.

Reply:
Not sure which config file but if the screen design changes now the code will also need to change. This can't be the only skill configurable value, can it?